Makefile

```diff
@@ -71,7 +71,7 @@ lint: license clean
 	taplo fmt --check
 	taplo lint
 	cargo clippy --all-targets -- -D warnings
-	cargo clippy --all-targets --no-default-features --features forest_deleg_cns,paritydb,instrumented_kernel -- -D warnings
+	cargo clippy --all-targets --no-default-features --features forest_deleg_cns,rocksdb,instrumented_kernel -- -D warnings
```

Comment:
I think `rocksdb` is a default feature so it'll be checked in the line above this one. Why do you need to add it here?

Reply:
Oh, I think I see. You don't want to compile the cli without the `rocksdb` feature. We should just remove the `rocksdb` and `paritydb` features from the cli. They don't belong there.

Reply:
It didn't and that's why the ci lint step was always red.

Reply:
Do you see what needs to be changed in the cli's Cargo file?